pkg/proxy/iptables_portalmanager.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
```

Comment:
I accept the review, but it will be next week before I get to this one.
Thanks for your patience :)

On Wed, Mar 25, 2015 at 4:47 PM, Robert Bailey <notifications@github.com>
wrote:

> In pkg/proxy/iptables_portalmanager.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5961#discussion_r27177860>
> :
>
> > @@ -0,0 +1,273 @@
> > +/*
> > +Copyright 2014 Google Inc. All rights reserved.
>
> New files should say 2015. You don't need to update the header in any
> files that you are modifying.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5961/files#r27177860>
> .
>